dvc/updater.py

```diff
@@ -41,7 +43,7 @@ def _with_lock(self, func, action):
                 func()
         except LockError:
             msg = "Failed to acquire '{}' before {} updates"
-            logger.debug(msg.format(self.lock.lock_file, action))
+            logger.debug(msg.format(self.lock.lockfile, action))
```

Comment:
Could've just left it as `lock_file` as it was (and this is the way we name such vars elsewhere) and avoided all these changes.

Reply:
This is what flufl lock uses. It would be strange to add `lock_file` prop proxying to `lockfile`.